pkg/cmd/provisioning/alibabacloud/create-ram-users_test.go

```diff
@@ -178,7 +178,7 @@ spec:
 
 	credReq := fmt.Sprintf(credReqTemplate, crName, targetSecretNamespace, targetSecretName)
 
-	f, err := ioutil.TempFile(targetDir, "testCredReq")
+	f, err := ioutil.TempFile(targetDir, "testCredReq*.yaml")
```

Comment:
can we keep `testCredReq.yaml` instead of `testCredReq*.yaml`. Is there any advantage of appending a random string here?

Reply:
It's about appending the random string in the right place. `ioutil.TempFile` always appends a random string as a suffix unless a `*` is present in the string in which case it substitutes `*` with the random string.

Created a Go Playground link to demonstrate this https://go.dev/play/p/EhARE5tzFCy